knowledge-base/setup-fiddler-alongside-outlook.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Setting Up Fiddler Everywhere alongside Outlook
+description: "Learn how to configure your proxy settings to bypass specific Microsoft endpoints so that you could use MS products like Outlook alongside Fiddler Everywhere."
+type: how-to
+slug: fiddler-and-outlook
+publish: true
+res_type: kb
+---
+
+## Environment
+
+|   |   |
+|---|---|
+| Fiddler Everywhere | 1.0.0 and above |
+| 3rd-party application | Microsoft Outlook, Office365 |
+| Operating System | Windows, macOS |
+
+## Description
+
+Some third-party applications like **Microsoft Outlook** are not working seamlessly alongside MITM proxies like Fiddler Everywhere.  How can I handle this issue?
```

Comment:
Use regular font for the product names, see https://testdocs.telerik.com/style-guide/element-formatting.